### ISSUE TITLE
Unstated Dependency on Job Scheduler

### ISSUE THREAD

**Issue by DarioS** [salzmanlab/MACHETE#2]
It seems an important omission that neither the journal article nor the README file state that SLURM is required to run the software. One reason software such as Bowtie 2 and limma is popular is because they don't put such barriers between themselves and their end-users. MACHETE seems like a bunch of disparate scripts that researchers outside of Stanford might need to battle with to get working in different computing environments.

**Comment by gillianhsieh** [comment]
Yes that is definitely true; thank you for bringing up this important issue.  We have ongoing work on a standalone version that does not require a scheduler.  I will addend the README now.
